src/services/resolver/peque-graphql.ts

```diff
@@ -56,40 +59,43 @@ export class PequeGraphQLService {
   #buildInterface(instance: InstanceType<ResolverDeclaration>, metadata: IResolverServiceMetadata): IResolvers {
     const resolver: IResolvers = {};
 
-    const objectAssign = (key: string, property: object): void => {
+    const objectAssign = (key: IQueryTypes | string, property: object): void => {
       Object.assign(resolver, { [key]: { ...resolver[key], ...property } });
     };
 
-    if (metadata.query) {
-      for (const query of metadata.query) {
-        const name = query.options?.name ?? query.method;
-        objectAssign('Query', {
-          [name]: async (parent, args, ctx, info) =>
-            await this.#buildMethodWithParams(instance, query.method)(parent, args, ctx, info),
-        });
-      }
-    }
-
-    if (metadata.field) {
-      for (const field of metadata.field) {
-        const type = this.#calculateType(field.options);
-        const name = field.options?.name ?? field.method;
-        objectAssign(type, {
-          [name]: async (parent, args, ctx, info) =>
-            await this.#buildMethodWithParams(instance, field.method)(parent, args, ctx, info),
-        });
+    const calculateMethods = (type: IQueryTypes): void => {
```

Comment:
This function seems hard to read. How would it look if we:

1. Return early if the `metadata[metadataType]` condition isn't met (should reduce nesting by 1 level)
2. Drop one-line ternary expression in favor of regular if statements
3. Drop `else` in favor of early return or `continue` in the for loop